packages/@ourworldindata/utils/src/owidTypes.ts

```diff
@@ -510,6 +510,32 @@ export type EnrichedBlockAside = {
     caption: Span[]
 } & EnrichedBlockWithParseErrors
 
+export const validChartShowKeywords = [
+    "all tabs",
+    "all controls",
+    "relative toggle",
+    "timeline",
+    "facet control",
+    "entity selector",
+    "country selector",
+    "zoom toggle",
+    "no data area toggle",
+    "data area toggle",
+    "area toggle",
+    "align axis scales toggle",
+    "align axis toggle",
+    "x log/linear selector",
+    "x log selector",
+    "y log/linear selector",
+    "y log selector",
+    "chart tab",
+    "map tab",
+    "table tab",
+    "download tab",
+] as const
```

Comment:
yeah, probably good to split them up!

Reply:
I'm a bit torn on this - I find the separation clearer if tabs are separate but it might be that very soon we'll change the visual UI elements of grapher so that the things that are currently all tabs are no longer one such clearly delinieated group. It probably doesn't matter too much though so I'm fine with both

Reply:
Regarding the format: In the spirit of GDocs and Archie, I wanted to give it a natural language feel. I agree though that, as a developer, kebap-casing would also make me feel more comfortable. I guess what matters in the end is a certain consistency across gdoc blocks, so maybe @ikesau or @danyx23 have thoughts?

Reply:
Oh, I didn't see your comment, Daniel, when I posted mine!

Reply:
I just changed `show` to `tabs` and `controls` – if you're fine with both, I'll keep them separated